tests/lcs/core/interface/image-undo.livecodescript

```diff
@@ -36,6 +36,12 @@ on TestFlipImageWorks
 end TestFlipImageWorks
 
 on TestClickImageWorks
+
+    if the environment is "command line" then
```

Comment:
This sounds like something we need to fix - can you file a bug about this? It should be possible to simulate a click in -ui mode so something must be preventing it from working for no good reason.